### PR TITLE
Remove diesel_codegen_shared

### PR DESCRIPTION
**Pull request by weiznich**
This moves the remaining functionality from diesel_codegen_shared to diesel_codegen and removes diesel_codegen_shared.

### CODE REVIEW

**Comment by killercup** [comment]
Oh. This kinda got hit by the MySQL stuff that was merged this week. Can you rebase?

**Comment by weiznich** [comment]
@killercup Rebase is done

File: diesel_codegen/Cargo.toml [hunk 1]
```diff
@@ -11,17 +11,21 @@ keywords = ["orm", "database", "postgres", "sql", "codegen"]
...
 [features]
 default = ["dotenv"]
-dotenv = ["diesel_codegen_shared/dotenv"]
-postgres = ["diesel_codegen_shared/postgres", "diesel_infer_schema/postgres"]
-sqlite = ["diesel_codegen_shared/sqlite", "diesel_infer_schema/sqlite"]
-mysql = ["diesel_codegen_shared/mysql", "diesel_infer_schema/mysql"]
+postgres = ["diesel_infer_schema/postgres"]
+sqlite = ["diesel_infer_schema/sqlite"]
+mysql = ["diesel_infer_schema/mysql"]
+
```
Comment (sgrif) [inline diesel_codegen/Cargo.toml]: ✂️

Comment (weiznich) [inline diesel_codegen/Cargo.toml]: fixed

**Pull request merged**